src/next-steps/sub-components/task.tsx

```diff
@@ -76,26 +76,28 @@ export function Task( { taskId }: Props ) {
 
 	let taskIsBroken = false;
 	let titleDisplay: ReactNode;
+	let buttonDisplay: ReactNode = null;
 	if ( link ) {
 		const handleLinkClick = () => {
 			monitoringClient.analytics.recordEvent( 'task_link_click', { linkType: link.type } );
 			handleCheckboxChange();
 		};
 
 		try {
-			const linkText = title || getDefaultTitleForLink( link );
+			const titleText = title || getDefaultTitleForLink( link );
 			const href = createLinkHref( link, issueTitle );
-			titleDisplay = (
+			titleDisplay = <span className={ styles.taskTitle }>{ titleText }</span>;
+			buttonDisplay = (
 				<a
-					className={ styles.taskTitle }
+					className={ styles.taskLink }
 					target="_blank"
 					href={ href }
 					rel="noreferrer"
 					// When they open a link, let's trigger the checkbox change too
 					onClick={ handleLinkClick }
 				>
 					{ getAppIconForLink( link ) }
-					<span className={ styles.linkText }>{ linkText }</span>
+					<span className={ styles.linkText }>{ getLinkName( link ) }</span>
 					<LinkIcon aria-hidden={ true } className={ styles.linkIcon } />
 				</a>
```

Comment:
**Suggestion (non-blocking)**: Re: accessibility, I think what is here is fine, but having the link only say the link title on a screen reader feels a little terse, and could maybe use a bit of context!

I wonder if using `aria-describedby` and using the issue title might be the best way forward?

Here is what that could look like:

``` typescript
const titleId = removeSpaces(`title-${ taskId }`);
titleDisplay = <span id={titleId} className={ styles.taskTitle }>{ titleText }</span>;
buttonDisplay = (
<a
	className={ styles.taskTitle }
	className={ styles.taskLink }
	target="_blank"
	href={ href }
	rel="noreferrer"
        aria-describedby={titleId}
	// When they open a link, let's trigger the checkbox change too
	onClick={ handleLinkClick }
>
	{ getAppIconForLink( link ) }
	<span className={ styles.linkText }>{ linkText }</span>
	<span className={ styles.linkText }>{ getLinkName( link ) }</span>
	<LinkIcon aria-hidden={ true } className={ styles.linkIcon } />
</a>
```

That way, the screen reader will read: "Link: Github. <The title>" which seems nice to me!

Reply:
Thanks for your suggestion! It was interesting to learn how to implement better accessibility! I made my changes here: [adcffec](https://github.com/Automattic/bugomattic/pull/71/commits/adcffeca79cf0e8744a0dd030778d702310c57c5)